server/models/UserTwoFactorMethod.ts

```diff
@@ -71,6 +71,7 @@ export default class UserTwoFactorMethod<
       group: 'method',
       where: {
         UserId: userId,
+        deletedAt: null,
```

Comment:
I assumed that the the `deletedAt: null` would be redundant because the model is set to be paranoid, is that not the case?

https://github.com/opencollective/opencollective-api/blob/543812902d702526abf9d473b2c131f0425dc4af/server/models/UserTwoFactorMethod.ts#L178

Reply:
Indeed, the `deletedAt` condition is automatically added by default on paranoid models.